fe/fe-core/src/main/java/org/apache/doris/planner/PlanNode.java

```diff
@@ -133,7 +134,7 @@ abstract public class PlanNode extends TreeNode<PlanNode> {
 
     protected List<SlotId> outputSlotIds;
 
-    private NodeType nodeType = NodeType.DEFAULT;
+    protected NodeType nodeType = NodeType.DEFAULT;
     protected StatsDeriveResult statsDeriveResult = new StatsDeriveResult();
 
     protected PlanNode(PlanNodeId id, ArrayList<TupleId> tupleIds, String planNodeName) {
```

Comment:
```suggestion
    protected PlanNode(PlanNodeId id, ArrayList<TupleId> tupleIds, String planNodeName, NodeType nodeType) {
```

Reply:
In order to reduce file modifications, this PR only makes special modifications to SCAN_NODE, and will subsequently initialize Node as a parameter